instance/static/html/instance/index.html

```diff
@@ -58,12 +65,22 @@
               </li>
             </ul>
           </div>
-          <div ng-switch="selected.instance.status">
-            <button class="button reprovision" ng-switch-when="ready"
-                                               ng-click="provision(selected.instance)">
-              Reprovision
-            </button>
-            <img src="/static/img/running.png" tooltip-placement="bottom" tooltip="Provisioning in progress..." alt="Provisioning in progress..." ng-switch-default />
+          <div class="large-2 columns">
+            <div class="status-icon" ng-switch="selected.instance.status">
+              <button class="button reprovision" ng-switch-when="ready"
+                                                 ng-click="provision(selected.instance)">
+                Reprovision
+              </button>
+              <button class="button reprovision" ng-switch-when="error"
+                                                 ng-click="provision(selected.instance)">
+                Reprovision
+              </button>
```

Comment:
I think this ng-swtich block should all be replaced with a ng-if, like this:

    ng-if="selected.instance.status == 'ready' || selected.instance.status == 'new' || selected.instance.status == 'empty'

Or even better write a controller method:

    $scope.can_be_provisioned = function() {
        return $scope.selected.instance.status != ...
    }

Thoughts?